### ISSUE TITLE
Namespaced policy not found

### ISSUE THREAD

**Issue by pablocrivella** [palkan/action_policy#2]
Current scenario:

```ruby
# models/appointment.rb
class Appointment
end

# policies/care/appointment_policy.rb
module Care
  class AppointmentPolicy < ApplicationPolicy
  end
end

# controllers/care/appointments_controller.rb
module Care
  class AppointmentsController < BaseController
    def index
      authorize!
    end
  end
end

# => ActionPolicy::NotFound in Care::AppointmentsController#index 
# => Couldn't find policy class for Appointment
```

Ruby version: 2.5.0
Rails: 5.2.0

Errors seems to be that when this check is made: 

https://github.com/palkan/action_policy/blob/daa2d12afdf2f4ba47a231c50e749f20c0f7e6ad/lib/action_policy/lookup_chain.rb#L36

The `Care` namespace only contains this constants:

```
mod.constants
=> [:AppointmentsController, :BaseController]
```

The weirdest thing is that my test are working:

```ruby
# frozen_string_literal: true

require "rails_helper"

RSpec.describe "GET /care/appointments", type: :request do
  before { sign_in(user) }

  subject { get care_appointments_path }

  context "as a caretaker" do
    let(:user) { create(:caretaker, :confirmed) }

    it "is authorized" do
      expect { subject }.to be_authorized_to(:index?, Appointment)
        .with(Care::AppointmentPolicy)
    end
  end
end
```

Any idea what could be the issue here?

**Comment by palkan** [comment]
Hi, @pablocrivella!

> The weirdest thing is that my test are working:

That could be a clue. Do you have `config.eager_load = true` in your `environments/test.rb`? And `config.eager_load = false` in your `development.rb` respectively?

**Comment by pablocrivella** [comment]
@palkan It's false on both. Should those settings be "true" for this to work?

**Comment by palkan** [comment]
>  Should those settings be true for this to work?

Yep, that should help.

I've pushed a fixed (hopefully); please, try `master` branch.

**Comment by palkan** [comment]
@pablocrivella Thanks!

Released as 0.1.1
